public/components/TagEdit/formComponents/TagName.react.js

```diff
@@ -223,7 +223,7 @@ export default class TagNameEdit extends React.Component {
             <div className={classNames.slug.lock} onClick={this.toggleSlugLock.bind(this)}></div>
             <label>Path</label>
             <div className="tag-edit__linked-field__input-container">
-              <span>{!this.props.pathLocked ? this.getPathPrefixForSection() : this.props.tag.path}</span>
+              <span>{!this.props.pathLocked ? this.getPathPrefixForSection() : <a href={`https:\/\/theguardian.com/${this.props.tag.path}`} target="_blank">{this.props.tag.path}</a>}</span>
```

Comment:
Can we add the `noreferrer` and `noopener` rels please? https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
Okie!

Reply:
PR open!